presto-native-execution/presto_cpp/main/PrestoServer.h

```diff
@@ -264,6 +269,7 @@ class PrestoServer {
   folly::Synchronized<bool> shuttingDown_{false};
   std::chrono::steady_clock::time_point start_;
   std::unique_ptr<PeriodicTaskManager> periodicTaskManager_;
+  std::unique_ptr<PeriodicMemoryChecker> periodicMemoryChecker_;
```

Comment:
@minhancao let's remove this field. 
`folly::Singleton<PeriodicMemoryChecker>::try_get()` should suffice.
I spoke to Xiaomeng and we are okay with this.

Reply:
Done